provider/src/main/kotlin/org/gradle/kotlin/dsl/accessors/CodeGenerator.kt

```diff
@@ -22,142 +22,142 @@ import org.jetbrains.kotlin.lexer.KtTokens
 
 internal
 fun ProjectSchema<TypeAccessibility>.forEachAccessor(action: (String) -> Unit) {
-    extensions.forEach { (name, type) ->
-        extensionAccessorFor(name, type)?.let(action)
+    extensions.map(::typedAccessorSpecFor).forEach { spec ->
```

Comment:
:x: The suffix `For` in `typedAccessorSpecFor` doesn't make much sense from the perspective of the caller here as it's being passed by reference. `extensions.map(::typedAccessorSpec).forEach { ...` reads a little better.